src/Orckestra.Composer.Website/App_Code/RecurringOrderHelpers.cshtml

```diff
@@ -0,0 +1,115 @@
+﻿@using Composite.AspNet.Razor
+@using System.Web.Mvc
+
+@helper CartItemRecurringOrderProgram(string controllerName)
+{
+    <div data-oc-controller="@controllerName" 
```

Comment:
move data-oc-controller outside this helper, you can have here parameter "vueBinding" = "item" by default
```
<div data-oc-controller="SingleCheckout.ChangeRecurringFrequency" >
 @RecurringOrderHelpers.CartItemRecurringOrderProgram("itme")
</div>
```